x-pack/plugins/canvas/public/components/workpad_page/workpad_interactive_page/interaction_boundary.tsx

```diff
@@ -56,7 +56,7 @@ export class InteractionBoundary extends PureComponent<void, State, void> {
       {
         top: '50%',
         left: '50%',
-        position: 'absolute',
+        position: 'absolute' as 'absolute',
```

Comment:
I don't understand why this was necessary.  I reverted it locally with no issue...?

This is the only change for Canvas.  I can push the revert to the branch?

Reply:
Actually, I take that back... and there are more type issues than are listed here.  Update coming.